### DISCUSSION TITLE
Windows 11 PXE Boot

### DISCUSSION THREAD

**Discussion by cvoiculescu** [antonym/netboot.xyz#1164] [General] [answered]
Hello,

I added the WinPE image as described in documentation but I have issues installing Windows 11 with Setup.
It seems somehow while running the setup, the system is not recognized as compatible with Windows 11 (I guess it does not identify the TPM)
Is there any way to bypass this check, or maybe is there any other WinPE image to be used?

I looked over Internet and I was not able to find any documentation on this.

Thanks

**Comment by cvoiculescu** [comment, marked as answer]
Actually, I found out what is required
When creating the WinPE image, WinPE-WMI and WinPE-SecureStartup optional components has to be added
I tried, and seems everything is working now

**Reply by pauleec** [reply]
Can you provide the detail?

**Reply by m-assaleh** [reply]
For Windows 11: If you're launching Windows Setup from WinPE, make sure your WinPE image includes the [WinPE-WMI and WinPE-SecureStartup optional components](https://learn.microsoft.com/en-us/windows-hardware/manufacture/desktop/winpe-add-packages--optional-components-reference?view=windows-11#winpe-optional-components). If you don't include these optional components, you may see an error that your PC doesn't meet the minimum hardware requirements.

[Source](https://learn.microsoft.com/en-us/windows-hardware/manufacture/desktop/winpe-intro?view=windows-11#notes-on-running-windows-setup-in-windows-pe)

**Reply by cvoiculescu** [reply]
Indeed. This is the way. The only issue is that this makes the image much bigger. so unless you create an image for windows 11, don't do this as for all booting images
